data_generator_tests.py

```diff
@@ -56,19 +56,25 @@ def display_img(annId):
   plt.show()
 
 # %% Test the generator
+import pandas as pd
+import data_augmentation
+imp.reload(data_augmentation)
 import data_generator
 imp.reload(data_generator)
 from data_generator import DataGenerator
 
+
+representative_set_df = pd.read_pickle(os.path.join(DEFAULT_PICKLE_PATH, 'representative_set.pkl'))
 generator = DataGenerator(
-    df=val_df,
+    df=representative_set_df,
     base_dir=DEFAULT_TRAIN_IMG_PATH,
```

Comment:
Does `TRAIN` work here? (I think they're the same as `VAL`, right?)

Reply:
They're the same but Imma change to  `VAL_IMG_PATH ` b/c it's more clear